libraries/DataFlash/DataFlash.h

```diff
@@ -75,7 +75,6 @@ class DataFlash_Class
     void EraseAll();
 
     // possibly expensive calls to start log system:
-    bool NeedPrep();
     void Prep();
```

Comment:
Nitpicking: does the Prep still needs to be public? I would say it could even be removed and the one liner incorporated into *init*.